README.md

```diff
@@ -34,6 +34,12 @@ Download the script [here](https://github.com/js-cookie/js-cookie/blob/latest/sr
 <script src="/path/to/js.cookie.js"></script>
 ```
 
+Or include it via [jsDelivr CDN](https://www.jsdelivr.com/package/npm/js-cookie):
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.js"></script>
```

Comment:
Do you think it's worth adding the minified file instead as per @MartinKolarik's comment?

Reply:
Yes I think its worth it, so I replaced it with the minified file.